frame/support/test/tests/pallet_ui/pass/dev_mode_valid.rs

```diff
@@ -19,6 +19,16 @@ pub mod pallet {
 	#[pallet::storage]
 	type MyStorage<T: Config> = StorageValue<_, Vec<u8>>;
 
+	// The Hasher requirement skipped by `dev_mode`.
+	#[pallet::storage]
+	type MyStorageMap<T: Config> = StorageMap<_, _, u32, u64>;
```

Comment:
Thanks for the UI test.  
Could you please also add a test to verify that it works as expected?  
So to test that the correct hasher is used? Not sure if this is accessible from the metadata, otherwise you can use a `storage_alias` and compare the resulting keys.

Reply:
Done.